tests/unit/utils.test.ts

```diff
@@ -20,8 +20,8 @@ describe("Format functions", () => {
 
   test("formatRadicleId", () => {
     if (process.env.HEARTWOOD) {
-      expect(utils.formatRadicleId("rad:zKtT7DmF9H34KkvcKj9PHW19WzjT")).toEqual(
-        "rad:zKtT7D…19WzjT",
+      expect(utils.formatRadicleId("zKtT7DmF9H34KkvcKj9PHW19WzjT")).toEqual(
+        "zKtT7D…19WzjT",
```

Comment:
Hmm I'm not sure why we're changing this. The above is a valid radicle id in urn format. We also need to specify whether we're talking about repository ids or public keys..

Reply:
Interesting, yeah I think we need some design spec on all the identifiers.

Current state AFAIK
```
did:key:<public key>
Returned e.g.from delegate listings, seems more full format

rad:<public key>
Is this for projects only? Why do users don't have rad: prefixes?

<public key>
Defined as PublicKey or ActorId in httpd
```

I'm also ok with maybe a more general format function that can hande different identifier formats